CONTRIBUTING.md

```diff
@@ -2,42 +2,34 @@
 
 ## How to contribute
 
-If you would like to contribute to this project, see [Contributions to Splunk][indivcontrib] for more information.
+If you would like to contribute to this project, see [Contributions to Splunk](https://www.splunk.com/en_us/form/contributions.html) for more information.
 
-## Issues & Bug Reports
+## Issues and bug reports
 
-If you're seeing some unexpected behavior with this project, please create an [issue on GitHub][issues] with the following information:
+If you're seeing some unexpected behavior with this project, please create an [issue](https://github.com/splunk/splunk-sdk-javascript/issues) on GitHub with the following information:
 
 1. Version of this project you're using (ex: 1.7.1)
-1. Platform version (ex: Windows Server 2012)
-1. Framework version (ex: Node.js 0.10.37) or Browser (ex: Chrome 43.0.2357.81)
-1. Splunk version (ex: 6.2.2)
-1. Other relevant information (ex: local/remote environment, Splunk network configuration)
+2. Platform version (ex: Windows Server 2012)
+3. Framework version (ex: Node.js 0.10.37) or Browser (ex: Chrome 43.0.2357.81)
```

Comment:
Tested Node versions are 0.12, 4.2, and 10.0 - we should probably list one of those

Reply:
got it